docker-compose.yml

```diff
@@ -1,10 +1,11 @@
-version: '3'
+version: "3"
 
 services:
   mindwendel:
     container_name: mindwendel
     build:
       context: .
+      target: development
```

Comment:
what do you think about using env variables here instead? to make it more clear that you have to adapt it to your needs, e.g. using a prod build.

see https://docs.docker.com/compose/environment-variables/

Reply:
i would also advise to add the prod target to the readme. Otherwise people will start this in dev mode...

Reply:
Based on you feedback, I adjusted the `docker-compose.yml` to avoid starting mindwendel in dev mode. The default `docker-compose.yml` pulls the published mindwendel prod image. I also added documentation in the `README.md` .

For development purposes, I renamed the existing `docker-compose.yml` to `docker-compose.dev.yml` that can be used for development purpose only. I also added a section in the `README.md`.